src/test/java/com/hocs/test/glue/GenericInputStepDefs.java

```diff
@@ -103,7 +126,16 @@ public void pageTitleIs(String title) {
         page.assertTitle(title);
     }
 
-    @Then("^I see the \"([^\"]*)\" message$")
+    @Then("^I should send the string of the first child to the console$")
```

Comment:
this kind of method shouldnt really exist, if you want to put the first child to the console, do it at that stage in the test ? also output this to console will only help with debugging so it shouldnt always be done? consider taking this out ?